packages/nocodb/src/modules/jobs/migration-jobs/nc_job_004_cleanup_duplicate_column.ts

```
@@ -0,0 +1,20 @@
+import debug from 'debug';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class CleanupDuplicateColumnMigration {
+  private readonly debugLog = debug(
+    'nc:migration-jobs:cleanup-duplicate-column',
+  );
+
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)